arch/sim/src/sim/sim_alsa.c

```diff
@@ -116,6 +177,32 @@ static const struct audio_ops_s g_sim_audio_ops =
   .release       = sim_audio_release,
 };
 
+static const struct sim_codec_ops_s g_mp3_dec =
+{
+  AUDIO_FMT_MP3,
+  AUDCODEC_DEC,
+  sim_audio_mp3_init,
+  sim_audio_mp3_samples,
+  sim_audio_mp3_decode,
+  sim_audio_mp3_uninit
+};
+
+static const struct sim_codec_ops_s g_pcm_codec =
+{
+  AUDIO_FMT_PCM,
+  AUDCODEC_DEC | AUDCODEC_ENC,
+  sim_audio_pcm_init,
+  NULL,
+  sim_audio_pcm_process,
+  sim_audio_pcm_uninit,
+};
+
+static const struct sim_codec_ops_s g_codec_s[] =
+{
+  g_mp3_dec,
```

Comment:
let' inline declare element to avoid the dup

Reply:
hi pkarashchenko,
i've got what you said, and fixed.
thanks.